kotlinx-coroutines-core/common/src/internal/CoroutineExceptionHandlerImpl.kt

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2016-2022 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.coroutines.internal
+
+import kotlinx.coroutines.*
+import kotlin.coroutines.*
+
```

Comment:
We have a different naming scheme for expect/actuals: common files are named as `Foo.common.kt`, while platform-specific files are named `Foo.kt`.

Apart from consistency, it will help to reduce the diff and make historic git-digging much easier